Jenkinsfile_k8s

```diff
@@ -1,4 +1,17 @@
+
+// The production credentials are only used on the main branch, when the build is triggered by code push or a human
+// On branches, PR, or the daily rebuild, then tests are run, no deployment are executed, and the "non production" credentials are used
+def CREDENTIAL_BASE_ID = 'ci-terraform'
+if (env.BRANCH_NAME == 'main' && !currentBuild.getBuildCauses('hudson.triggers.TimerTrigger$TimerTriggerCause')) {
+  CREDENTIAL_BASE_ID = 'production-terraform'
+}
+
 pipeline {
+  triggers {
+    // Build once per day if not a Pull Request or not a tag build
+    cron(env.CHANGE_ID || env.TAG_NAME ? '' : 'H 1 * * *')
```

Comment:
Wouldn't it be better to use H H * * *

Reply:
I've copy-and-pasted from a pipeline of mine, and I realize that we can even use the `@daily` notation to make it easier to read (because cron syntax is the not the most user friendly syntax of this world...). Thanks!